contracts/pm/mixins/MixinReserve.sol

```diff
@@ -1,29 +1,22 @@
-pragma solidity ^0.5.11;
-// solium-disable-next-line
-pragma experimental ABIEncoderV2;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.8;
 
+import "./MixinContractRegistry.sol";
 import "./interfaces/MReserve.sol";
-import "./interfaces/MContractRegistry.sol";
-import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 
-contract MixinReserve is MContractRegistry, MReserve {
+abstract contract MixinReserve is MixinContractRegistry, MReserve {
```

Comment:
Thoughts on continuing to inherit from MContractRegistry instead so this abstract contract doesn't need to be aware of the impl of the functions in MixinContractRegistry and those implementations would just be mixed in via the MixinContractRegistry mixin in TicketBroker? Mainly interested in whether we can minimize the diff here if this change is not needed.